api/mesh/v1alpha1/retry.proto

```diff
@@ -56,6 +56,22 @@ message Retry {
 
       //  +optional
       repeated uint32 retriable_status_codes = 5;
+
+
+      enum Method {
```

Comment:
See the Method enum in gateway_route.proto. I think that the right approach here is to host the Method enum out of gateway_route.proto into a new protobuf file, then to use that definition in both places.

Please use the definition from gateway_route.proto, since it's zero value is defined unambiguously.

Reply:
I have define define http_method type in a new proto file and both gateway_route and retry reference it.